### PR TITLE
Add Ping Frame send capability

### PR DESCRIPTION
**Pull request by CharlesAHunt**
Add Ping Frame send capability to resolve issue https://github.com/andyglow/websocket-scala-client/issues/208

### CODE REVIEW

**Comment by andyglow** [comment]
Hi, @CharlesAHunt 
thank you for the contribution

may I ask you to remove `final` keywords for now? just in case.. as we don't really know how the library is getting used and if there is somebody who somehow extends those classes

File: src/main/scala/com/github/andyglow/websocket/WebsocketClient.scala [hunk 1]
```diff
@@ -90,7 +90,7 @@ object WebsocketClient {
 
   def apply[T: MessageAdapter](uri: Uri)(receive: PartialFunction[T, Unit]): WebsocketClient[T] = apply(uri, WebsocketHandler(receive))
 
-  case class Builder[T: MessageAdapter](
```
Comment (andyglow) [inline src/main/scala/com/github/andyglow/websocket/WebsocketClient.scala:93]: please don't make them `final`

**Review by andyglow — CHANGES_REQUESTED** [review]
looks good. thank you
but please don't add `final` keywords yet

File: src/main/scala/com/github/andyglow/websocket/WebsocketClient.scala (hunk 1 above)
Comment (CharlesAHunt) [reply src/main/scala/com/github/andyglow/websocket/WebsocketClient.scala]: @andyglow Removed all the finals. Thanks for reviewing!

**Review by andyglow — APPROVED** [review]
lgtm. thank you

**Pull request merged**